python/action-server/action.yml

```diff
@@ -0,0 +1,8 @@
+base_key_expr: Genotyper/1/DNASensor/1
+iter: 10
+target: ALL
+config: 
+listen: 
```

Comment:
why?

Reply:
not sure if most of these are actually needed. Kindly explain if you have reason(s)

Reply:
iter and target are not necessarily needed so I removed them because they are used in the pub and get respectively which is not necessarily needed.